src/hooks/useDrag.tsx

```diff
@@ -184,6 +184,31 @@ export const useDrag: OnDragEndResponseWrapper = (
   return updateHomepageState(result, homepageState)
 }
 
+type UpdateHomepageType =
```

Comment:
instead of defining a separate type and needing to maintain this + `HomepageDroppableZone`, could we instead just use the latter since **they should be the same**

Reply:
They're not :(

HomepageDroppableZone is 
```
type HomepageDroppableZone =
  | "dropdownelem"
  | "leftPane"
  | "highlight"
  | "announcement"
  | `textCardItem-${number}`
```
UpdateHomepageType is
```
type UpdateHomepageType =
  | "editor"
  | "dropdownelem"
  | "highlight"
  | "announcement"
  | `textCardItem-${number}`
```

Reply:
sgtm